nutanix/resource_nutanix_virtual_machine.go

```diff
@@ -9,7 +9,7 @@ import (
 	"strings"
 	"time"
 
-	"github.com/terraform-providers/terraform-provider-nutanix/client/v3"
+	v3 "github.com/terraform-providers/terraform-provider-nutanix/client/v3"
```

Comment:
Isn't this import by default `v3` without the name? Why using a named import?

Reply:
Is the way that go 1.12 version do the goimports